project/Common.scala

```diff
@@ -118,6 +117,8 @@ object Common extends AutoPlugin {
     // timeout.
     Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-F", "4"),
     scalafmtOnCompile := false,
-    javafmtOnCompile := false,
-    sonatypeProfileName := "org.apache.pekko")
+    javafmtOnCompile := false)
+
+  override lazy val buildSettings = Seq(
+    dynverSonatypeSnapshots := true)
```

Comment:
@pjfanning This is what was causing the issue. If you dont use sbt-ci-release you need to set this value to true so that `dynverSonatypeSnapshots` sets the project version to a snapshot unless you are at a commit with a git tag.

The reason why it was publishing locally is that sbt-sonatype was actually creating a local bundle which is whats used when doing a release publish and this was happening because without `dynverSonatypeSnapshots` the projects didn't have the `-SNAPSHOT`